### PR TITLE
[helm chart] replace name with fullname helper

### PR DESCRIPTION
**Pull request by rekcah78**
## Motivation

Value fullnameOverride can't be used

## Changes
**Helm Chart**
* replace name helper with the fullname helper
* add default labels for helm chart

## Tests done

* Deploying with and without full name replacement

## TODO

- [x] I've assigned myself to this PR

### CODE REVIEW

**Review by JTaeuber — CHANGES_REQUESTED** [review]

**Comment by JTaeuber** [comment]
Other than that these changes are good. If you made the changes and rebased the branch so it includes the changes of https://github.com/caas-team/py-kube-downscaler/pull/78 we can merge this.

File: chart/templates/rbac.yaml
[the remark this answers is not in the document]
Comment (rekcah78) [reply chart/templates/rbac.yaml:1]: done

File: chart/templates/_helpers.tpl
[the remark this answers is not in the document]
Comment (rekcah78) [reply chart/templates/_helpers.tpl:1]: done

**Comment by JTaeuber** [comment]
Okay perfect, now we just need a rebase to resolve the current merge conflicts.

**Review by JTaeuber — APPROVED** [review]
LGTM

**Pull request merged**